packages/clay-management-toolbar/package.json

```diff
@@ -0,0 +1,34 @@
+{
+	"name": "@clayui/management-toolbar",
+	"version": "3.0.0",
+	"description": "ClayManagementToolbar component",
+	"license": "BSD-3-Clause",
+	"repository": "https://github.com/liferay/clay/tree/master/packages/clay-management-toolbar",
+	"engines": {
+		"node": ">=0.12.0",
+		"npm": ">=3.0.0"
+	},
+	"main": "lib/index.js",
+	"types": "lib/index.d.ts",
+	"ts:main": "src/index.tsx",
+	"files": ["lib", "src"],
+	"scripts": {
+		"build": "cross-env NODE_ENV=production babel src --root-mode upward --out-dir lib --extensions .ts,.tsx",
+		"build:types": "cross-env NODE_ENV=production tsc --project ./tsconfig.declarations.json",
+		"prepublishOnly": "npm run build && npm run build:types",
+		"test": "jest --config ../../jest.config.js"
+	},
+	"keywords": ["clay", "react"],
+	"dependencies": {
+		"classnames": "^2.2.6",
+		"@clayui/label": "^3.0.0-milestone.1"
```

Comment:
Should this be milestone.2 now?

Reply:
Yeah, I had started working on it before `milestone.2`. Thanks